github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -266,3 +266,41 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- name: periodic-kubevirt-mirror-uploader
+  cluster: ibm-prow-jobs
+  cron: "05 7 * * *"
+  max_concurrency: 1
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: fgimenez
+    repo: project-infra
+    base_ref: add-git-pr-script
+    workdir: true
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+  spec:
+    containers:
+      - image: "kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab"
+        env:
+        - name: GIT_ASKPASS
+          value: "hack/git-askpass.sh"
+        command:
+          - "hack/git-pr.sh"
+          - "-c"
+          - "bazelisk run //plugins/cmd/uploader:uploader -- -workspace ../kubevirt/WORKSPACE"
```

Comment:
I don't think that we have bazelisk installed in the builder image (the image contains bazel only at the moment). Running `go get -u github.com/bazelbuild/bazelisk` may solve that.

Reply:
Thx @rmohr following our online conversation i've pushed changes to use the bazel version present in the image, which is the required one.

Reply:
Finally the bazel version installed in the image doesn't match the required, pushed changes to install it as suggested